apps/nextjs-website/src/app/ThemeRegistry.tsx

```diff
@@ -0,0 +1,75 @@
+'use client';
+import { ReactNode, useState } from 'react';
+import createCache from '@emotion/cache';
+import { useServerInsertedHTML } from 'next/navigation';
+import { CacheProvider } from '@emotion/react';
+import CssBaseline from '@mui/material/CssBaseline';
+import { theme as muiItaliaTheme } from '@pagopa/mui-italia';
+import { createTheme, ThemeProvider } from '@mui/material';
+import { Options } from '@emotion/cache/dist/declarations/types';
+
+export const theme = createTheme(muiItaliaTheme);
+
+// This implementation is from emotion-js
+// https://github.com/emotion-js/emotion/issues/2928#issuecomment-1319747902
```

Comment:
This comment should be removed once [this issue](https://pagopa.atlassian.net/jira/software/projects/DEV/boards/280/backlog?selectedIssue=DEV-955) is completed